pt-br/index.html

```diff
@@ -0,0 +1,502 @@
+            Pessoas com deficiência não usam a web
+          </h2>
+
+          <p>
+            Isso esta errado. Na verdade, em alguns casos, a internet é 
+            <strong>a única maneira das</strong> pessoas com deficiência se 
+            comunicarem com o mundo. Além disso, "pessoas que não tem deficiências" 
+            usam seus sentidos naturais, como visão e audição,
+            <strong>pessoas com deficiência contam com tecnologias assistivas</strong>.
+          </p>
+
+          <a href="https://www.w3.org/WAI/people-use-web/" target="_blank"
+            >Mais detalhes
+            <span class="visually-hidden"
+              >sobre pessoas com deficiência não usam a web</span
+            ></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">Cegos não assistem filmes</h2>
+
+          <p>
+            Eles definitivamente assistem. Com ajuda de 
+            <strong>áudio descrições</strong> eles podem assistir e ouvir qualquer
+            conteúdo multimídia como todas as outras pessoas.
+          </p>
+
+          <a href="https://www.w3.org/WAI/media/av/description/" target="_blank"
+            >Mais detalhes
+            <span class="visually-hidden"
+              >sobre cegos não assistem filmes</span
+            ></a
+          >
+        </section>
+
+        <section class="myth">
+          <h2 class="myth__title">
+            Acessibilidade só pode ser testada por pessoas com deficiência
+          </h2>
+
+          <p>
+            Bem, geralmente as pessoas com deficiência são as melhores testadoras - 
+            Elas usam tecnologias assistivas em tempo integral. No entanto,
+            <strong>todos podem aprender</strong> a testar a acessibilidade de sites.
+          </p>
+
+          <a href="https://www.w3.org/WAI/test-evaluate/" target="_blank"
+            >Mais detalhes
+            <span class="visually-hidden"
+              >sobre acessibilidade só pode ser testada por pessoas com deficiência</span
+            ></a
+          >
+        </section>
+
+        <p>Mostre este site ao seu gerente.</p>
+      </div>
+    </main>
+
+    <footer>
+      <div class="content">
+        <p>
+          © 2021
+          <a target="_blank" href="https://www.sergeikriger.com/"
+            >Sergei Kriger</a
+          >. Todos os direitos reservados.
+        </p>
+        <p>
+          Construido por
+          <a target="_blank" href="https://en.wikipedia.org/wiki/Hand">Hands</a
+          >. Hospedado em
+          <a target="_blank" href="https://www.netlify.com/">Netlify</a>. O
+          código fonte está no
+          <a target="_blank" href="https://github.com/sergeicodes/a11ymyths"
+            >GitHub</a
+          >. Acompanhe
+          <a target="_blank" href="https://twitter.com/_sergeikriger"
+            >@_sergeikriger</a
+          >. Translated by <a target="_blank" href="https://www.felipegenuino.com/"
```

Comment:
@felipegenuino could you translate this.

Reply:
Traduzido por